src/main/java/org/mockito/plugins/MockitoPlugins.java

```diff
@@ -41,4 +41,19 @@ public interface MockitoPlugins {
      * @since 2.10.0
      */
     MockMaker getInlineMockMaker();
+
+    /**
+     * Returns {@link MockMaker} instance used by Mockito with the passed name {@code mockMaker}.
+     *
+     * <p>This will return the instance used by Mockito itself, not a new instance of it.
+     *
+     * <p>This method can be used to increase the interop of mocks created by Mockito and other
+     * libraries using Mockito mock maker API.
+     *
+     * @param mockMaker the name of the mock maker or {@code null} to retrieve the default mock maker
+     * @return instance of the mock maker
+     * @throws IllegalStateException if a mock maker with the name is not found
+     * @since 5.6.0
+     */
+    MockMaker getMockMaker(String mockMaker);
```

Comment:
This interface can be implemented, meaning that we can break downstream projects. We should add a default implementation and probably also mark this interface with `@NotExtensible`